src/node_crypto.cc

```diff
@@ -270,7 +381,11 @@ void ThrowCryptoError(Environment* env,
       .ToLocalChecked();
   CryptoErrorVector errors;
   errors.Capture();
-  auto exception = errors.ToException(env, exception_string);
+  Local<Value> exception = errors.ToException(env, exception_string);
+  Local<Object> obj;
+  if (!exception->ToObject(env->isolate()->GetCurrentContext()).ToLocal(&obj))
```

Comment:
`env->context()`? If there's a specific reason for using GetCurrentContext(), you should mention it in a comment.